packages/app-web/src/views/app/app.tsx

```diff
@@ -68,13 +68,15 @@ export default function AppView (): JSX.Element {
 
   useWindowLoadListener(onAppLoad)
 
-  // Shortcut handler
+  /**
+   * Shortcut handler, receives shortcut notation from a function that calls it, this function looks up the shortcut notation in the shortcut "lookup" table and dispatches the action
+   */
```

Comment:
Please stick with the `//`-style comments within the body of functions and classes and wrap it at a line length of 80 characters (you may use the rulers setting in vscode for this).

Reply:
Ok, will do, I originally chose to use JSDoc originally so that as long as the function could be tracked all the way to it's source, VSCode will always show the associated JSDoc

Reply:
I see what you mean. Intuitively it looked strange to me to see it within the React function body. I prefer this over JSDoc in this context.